game/constants.ts

```diff
@@ -3,6 +3,6 @@ export const MAX_POTIONS = 99;
 export const RARITY_SCORE: Record<string, number> = { 'Common': 1, 'Uncommon': 2, 'Rare': 3 };
 export const CHOICE_SCORE_THRESHOLD = 10;
 export const BP_PER_ROOM = 10;
-export const DECK_SIZE = 32;
+export const DECK_SIZE = 18;
```

Comment:
We need to introduce an item deck size that stays at 32, differentiating from the room deck size.